### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

File: client/src/app/site/pages/organization/pages/accounts/services/common/account-filter.service.ts [hunk 1]
```diff
@@ -205,7 +205,7 @@ export class AccountFilterService extends BaseFilterListService<ViewUser> {
                 options: [
                     {
                         condition: [DuplicateStatus.All, DuplicateStatus.SameName],
-                        label: `Same first/last name`
+                        label: `Same given/surname`
```
Comment (Elblinator) [inline client/src/app/site/pages/organization/pages/accounts/services/common/account-filter.service.ts:208]:
```suggestion
                        label: `Same Sur- and Given name`
```

**Review by Elblinator — CHANGES_REQUESTED** [review]

File: client/src/app/site/pages/organization/pages/accounts/services/common/account-filter.service.ts (hunk 1 above)
Comment (emanuelschuetze) [reply client/src/app/site/pages/organization/pages/accounts/services/common/account-filter.service.ts]: Fixed

**Review by Elblinator — APPROVED** [review]

File: client/src/assets/i18n/de.po [hunk 2]
```diff
@@ -2948,12 +2939,21 @@ msgstr "Veranstaltungstitel"
 msgid "Meetings"
 msgstr "Veranstaltungen"
 
+msgid "Meetings effected:"
```
Comment (peb-adr) [inline client/src/assets/i18n/de.po:2942]: ~~effected~~
affected

noticed while scrolling through #3834 ... Maybe this will be remembered next time :P

**Review by peb-adr — COMMENTED** [review]
Small misspell

**Pull request merged**